test/library/standard/DateTime/readWriteDatesTimes.good

```diff
@@ -0,0 +1,3 @@
+datetime: true
+date: true
+time: true
```

Comment:
Do we have any tests that actually show the date-time formats in the .good file? Might be worth adding one so we can be sure it doesn't accidentally change.

Reply:
I'll add one.